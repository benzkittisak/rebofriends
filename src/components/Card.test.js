import { shallow , mount , render } from 'enzyme'; 
import Card from './Card';

describe('<Card/>', () => {
    test('expect to render Card component', () => {
        const wrapper = shallow(<Card/>)
        expect(wrapper.length).toEqual(1)
        expect(wrapper).toMatchSnapshot();
    })
    test('Should have a header ', () => {
        const wrapper = shallow(<Card/>)
        expect(wrapper.find('h2')).toHaveLength(1)
    })

    test('Should have a image', () => {
        const wrapper = shallow(<Card/>)
        expect(wrapper.find('img')).toHaveLength(1)
    })

    test('should have a props', () => {
        const wrapper = shallow(<Card/>)
        expect(wrapper.props()).not.toBeNull();
    })
    
});
