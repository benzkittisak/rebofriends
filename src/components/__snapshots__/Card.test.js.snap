// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card/> expect to render Card component 1`] = `ShallowWrapper {}`;
